pages/dkp/kommander/2.2/licensing/essential/index.md

```diff
@@ -0,0 +1,82 @@
+* **Cert Manager:** A Kubernetes add-on to automate the management and issuance of TLS certificates from various issuing sources.
+* **Cluster Autoscaler:** A component that automatically adjusts the size of a Kubernetes cluster so that all pods have a place to run and there are no unneeded nodes.
+
+## Built-in GitOps
+
+DKP Essential comes bundled with GitOps, an operating model for Kubernetes and other cloud native technologies, providing a set of best practices that unify Git deployment, management and monitoring for containerized clusters and applications. GitOps works by using Git as a single source of truth for declarative infrastructure and applications. With GitOps, the use of software agents can alert on any divergence between Git with what's running in a cluster, and if there's a difference, Kubernetes reconcilers automatically update or rollback the cluster depending on the case.
+
+## DKP Essential single cluster UI
+
+Bundled with DKP Essential is a single cluster management UI that can be used in lieu of the bundled CLI. From the UI you can:
+
+* **Setup identity providers:** DKP Essential supports GitHub, LDAP, SAML, and standard OIDC identity providers such as Google. These identity management providers support the login and authentication process for your Kubernetes cluster. See [Identity Providers][identityprov] for more information.
+* **Deploy applications:** The DKP Essential UI allows you to customize your workspace application deployments via the Applications page within the UI.
+* **Add a license:** To add a license via the UI, see [Add a license to Kommander][addlicense]
+
+For more information concerning the global and workspace-level UI, see [Workspaces][workspaces]
+
+## FAQ
+
+* I'm getting a denial message when I attempt to attach or manage clusters with my DKP Essential self-managed cluster, why?
+
+DKP Essential only provides for the ability to stand up an individual, self-managed kubernetes cluster. Attaching or managing other clusters via a management cluster is only available by upgrading to [DKP Enterprise][dkpenterprise]
+
+* I used to be able to work with one free attached cluster, and now I can't see it in the UI, why?
+
+Prior to DKP version 2.2 you could create or attach 1 additional cluster without a license. If you have a 2.1 cluster that utilized that 1 additional cluster and upgrade to 2.2, the upgrade will automatically put you into the DKP Essential UI, and you will no longer have access to that additional cluster in the UI. However, the additional cluster will still appear within the CLI. To resolve this, enter your [DKP Enterprise][dkpenterprise] license in either the UI or CLI, and the previously attached cluster and deployed applications will reappear within the UI.
```

Comment:
I think we should probably have a meeting about how to message this.  My understanding is that this issue is going to be discussed with the field teams next week, I think we should hold off until that meeting is over to understand how best to message this.